__tests__/api/BundleDeploy/BundleDeployer.test.ts

```diff
@@ -63,98 +67,146 @@ describe("BundleDeployer01", () => {
     });
     it("should complain if cicshlq not found", async () => {
 
-        jest.spyOn(ZosmfSession, "createBasicZosmfSession").mockImplementationOnce(() => ({}));
+        const createSpy = jest.spyOn(ZosmfSession, "createBasicZosmfSession").mockImplementationOnce(() => ({}));
+        const listSpy = jest.spyOn(List, "allMembers").mockImplementationOnce(() => { throw new Error( "Injected CICSHLQ error" ); });
         await runDeployTestWithError();
+
+        expect(createSpy).toHaveBeenCalledTimes(1);
+        expect(listSpy).toHaveBeenCalledTimes(1);
     });
     it("should complain if cicshlq not found2", async () => {
 
-        jest.spyOn(ZosmfSession, "createBasicZosmfSession").mockImplementationOnce(() => ({}));
-        jest.spyOn(List, "allMembers").mockImplementationOnce(() => ( { val1: "wibble"}));
+        const createSpy = jest.spyOn(ZosmfSession, "createBasicZosmfSession").mockImplementationOnce(() => ({}));
+        const listSpy = jest.spyOn(List, "allMembers").mockImplementationOnce(() => ( { val1: "wibble"}));
         await runDeployTestWithError();
+
+        expect(createSpy).toHaveBeenCalledTimes(1);
```

Comment:
I'm not sure these `toHaveBeenCalled` checks add much useful validation to test and I'm concern they'll just constrain any future implementation changes